package.json

```diff
@@ -57,5 +57,14 @@
   },
   "jest": {
     "setupTestFrameworkScriptFile": "<rootDir>setupTests.js"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "npm test"
+    }
+  },
+  "lint-staged": {
+    "*.js": ["eslint"]
```

Comment:
Could add --fix to auto fix errors on commit. Does lint-staged need to include jsx files too?

Reply:
Thanks!  The reason I don't the linter running over `jsx` files is that I don't *have* any `jsx` files.  I read that Facebook doesn't recommend using the `jsx` extension anymore (https://github.com/facebook/create-react-app/issues/87) so I've just been using `js` for everything.  On the other hand, AirBnB recommends the `jsx` extension, so it could go either way.  http://airbnb.io/javascript/react/

Similarly, I don't run `--fix` because I've got a few areas where eslint throws a warning (but not an error) that I don't want to change that could be changed by `--fix`.  Thanks very much for the thoughtful comments :)